### PR TITLE
handle composer not installed

### PR DESCRIPTION
**Pull request by ildyria**

### CODE REVIEW

File: bootstrap/simple-errors.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+function displaySimpleError($tite, $code, $message)
+{
+	$error_display = file_get_contents(__DIR__ . '/../simple_error_template.html');
+	$replacing = array(
+		'$tite' => $title,
```
Comment (d7415) [inline bootstrap/simple-errors.php]: Typo?

Comment (ildyria) [reply bootstrap/simple-errors.php]: Damn Sonarcloud is pretty good too. :sweat_smile:

File: index.php [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?php
+
+require __DIR__ . '/bootstrap/simple-errors.php';
+
+displaySimpleError('ROOT', 403, '<span class="important">This is the root directory and it MUST NOT BE PUBLICALLY ACCESSIBLE.</span><br>
```
Comment (d7415) [inline index.php:5]: Am I correct that the 403 is just for show? :(

Comment (ildyria) [reply index.php:5]: what do you mean ?

Comment (d7415) [reply index.php:5]: Does it return HTTP 403 somehow?

Comment (d7415) [reply index.php:5]: ( https://www.php.net/manual/en/function.http-response-code.php )

Comment (ildyria) [reply index.php:5]: done.

**Pull request merged**